src/api/app/helpers/webui/reportables_helper.rb

```diff
@@ -2,9 +2,13 @@
 module Webui::ReportablesHelper
   include Webui::WebuiHelper
 
+  def reportable_not_found(reportable_type:)
+    return "The reported #{reportable_type.blank? ? 'object' : reportable_type.downcase} does not exist anymore."
```

Comment:
How can `reportable_type` ever be not nil?

Reply:
I added the explanation to the commit message. Please take a look again.